lib/validators/modelValidator.ts

```diff
@@ -867,6 +898,29 @@ export class ModelValidator extends SpecValidator<SpecValidationResult> {
     if (!isValid) {
       operationResult.isValid = false
       operationResult.request.isValid = false
+
+      if (
+        requestValidationErrors !== undefined &&
```

Comment:
Can we use `!!requestValidationErrors` instead of complex judgement.

Reply:
For detail, see [link](https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript)